docs/app/Components/ComponentDoc/ComponentExamples.js

```diff
@@ -22,15 +22,18 @@ export default class ComponentExamples extends Component {
   renderMissingExamples = () => {
     const { name } = this.props
     return (
-      <Message icon='book' className='info'>
-        If there's no
-        <a
-          href='https://github.com/TechnologyAdvice/stardust/pulls'
-        > pull request </a>
-        open for <code>&lt;{name} /&gt;</code> examples, you should
-        <a
-          href='https://github.com/TechnologyAdvice/stardust/blob/master/CONTRIBUTING.md'
-        > contribute</a>!
+      <Message info icon>
+        <Icon name='book' />
+        <Message.Content>
+          If there's no
+          <a
+            href='https://github.com/TechnologyAdvice/stardust/pulls'
+          > pull request </a>
+          open for <code>&lt;{name} /&gt;</code> examples, you should
+          <a
+            href='https://github.com/TechnologyAdvice/stardust/blob/master/CONTRIBUTING.md'
+          > contribute</a>!
+        </Message.Content>
```

Comment:
Example of housekeeping, the previous usage of Message here threw warnings as we do not support it any longer.